src/app/Board.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Board extends Model
+{
+    protected $guarded = array('id');
+
+    public static $rules = array(
+        'person_id' => 'required',
+        'title' => 'required',
+        'message' => 'required'
+    );
+
+    public function getData()
```

Comment:
bladeで使うことを想定していると思うので、カスタムアクセサを利用した方が良いと思います。
https://readouble.com/laravel/7.x/ja/eloquent-mutators.html

また `public function getData` はメソッド名として汎用性が高すぎるので何を目的とするメソッドなのか命名を変更した方が良いかと思います
```suggestion
    public function getDisplayTitleAttribute()
```